gatsby-node.js

```diff
@@ -128,6 +134,20 @@ exports.createSchemaCustomization = ({ actions }) => {
       thumbnail: String
       thumbnailAltText: String
     }
+
+    type HomeJson implements Node {
+      tiles: [HomeJsonTile]
+    }
+
+    type HomeJsonTile {
+      title: String
+      body: String
+      iconName: String
+    }
+
+    type SiteSearchIndex implements Node {
+      index: SiteSearchIndex_Index
+    }
```

Comment:
Consider adding these gatsby-node definitions to `plugins/cms-content-plugin/gatsby-node.js` or creating your own separate plugin for them. I'd like to avoid an eventuality where the root gatsby-node.js ends up being 900 lines of code long because it has to describe every entity in the whole system. We can use custom plugins as a way to keep the app configurations organized.

Reply:
I have another branch where I started doing this, but forgot about it in the recent rush to finish. I'll create another branch to break up the gatsby-node file after QA can test the current stuff.